src/payment-medium.js

```diff
@@ -80,6 +83,32 @@ class PaymentMedium extends ExchangePaymentMedium {
       return Promise.resolve(output);
     });
   }
+
+  static getSellAccounts (api, quote) {
+    return PaymentAccount.getAll(api, quote).then(accounts => {
+      this._accounts = accounts;
+      return accounts;
+    });
+  }
+
+  static addBankAccount (api, obj) {
+    return PaymentAccount.add(api, obj).then(account => {
+      this._accounts.push(account);
+      return account;
+    });
+  }
+
+  deleteBankAccount (api, id) {
```

Comment:
I prefer a delete method on the bank account instance, rather than a helper like this. Unless it's a real pain for the frontend.